packages/components/icon-button/src/IconButton.stories.tsx

```diff
@@ -128,3 +130,14 @@ export const Link: StoryFn = _args => (
     </IconButton>
   </div>
 )
+
+export const Toggle: StoryFn = () => {
+  const [pressed, setPressed] = useState(false)
+  const toggle = () => setPressed(!pressed)
+
+  return (
+    <IconButton aria-label="Add to favorites" aria-pressed={pressed} onClick={toggle}>
```

Comment:
```suggestion
    <IconButton aria-label="Add to favorites" aria-pressed={pressed} onClick={toggle} design={pressed ? 'filled' : 'outlined'}>
```

Let me suggest to align the toggle demos aligned suggesting changing the design when toggling as an ideal situation. Changing outline-fill icons is not enough from my perspective.